bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/communication/RadioThermostatConnector.java

```diff
@@ -124,7 +125,8 @@ public String sendCommand(@Nullable String cmdKey, @Nullable String cmdVal, @Nul
         String postJson = cmdJson != null ? cmdJson : "{\"" + cmdKey + "\":" + cmdVal + "}";
 
         try {
-            Request request = httpClient.POST(buildRequestURL(resource));
+            Request request = httpClient.POST(buildRequestURL(resource)).timeout(REQUEST_TIMEOUT_MS,
+                    TimeUnit.MICROSECONDS);
```

Comment:
```suggestion
            Request request = httpClient.POST(buildRequestURL(resource)).timeout(REQUEST_TIMEOUT_MS,
                    TimeUnit.MILLISECONDS);
```

Reply:
Thanks, fixed

Reply:
Not yet fixed.

Reply:
Weird, don't know what happened, anyway fixed.